Open-Source-Education.md

```diff
@@ -2,5 +2,6 @@
 
 
 ## Computer Science
+* https://teachyourselfcs.com/ - Overview of various CS topics and links to the best resources for each. 
```

Comment:
Nice one here, thanks
A little nitpick though: Can we have this modified to reflect markdown formatting? kindly check examples of the other ones.